docs/install-nuget-client-tools.md

```diff
@@ -25,6 +25,9 @@ The [MSBuild CLI](reference/msbuild-targets.md) also provides the ability to res
 
 The two NuGet CLI tools are `dotnet.exe` and `nuget.exe`. See [feature availability](#feature-availability) for a comparison.
 
+* To target .NET Core or .NET Standard, use the dotnet CLI. The dotnet CLI is required for the SDK-style project format, which uses the [SDK attribute](/dotnet/core/tools/csproj#additions).
```

Comment:
Maybe we can mention SDK based projects again?
dotnet CLI and SDK projects can target .NET Framework.